script.js

```diff
@@ -1,12 +1,363 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+const currentState = {
```

Comment:
It is a nice idea to use a single object to keep track of all the states that you need.